pillar/dev/top.sls

```diff
@@ -22,6 +22,13 @@ base:
     - fastly-logging
     - firewall.fastly-logging
 
+  'codespeed':
+    - match: nodegroup
+    - firewall.codespeed
+    - codespeed
+    - secrets.postgresql-users.codespeed
```

Comment:
The SLS file for this doesn't seem to have been added to git

Reply:
interesting 🧐